amplipi/app.py

```diff
@@ -681,6 +685,35 @@ def create_app(mock_ctrl=None, mock_streams=None, config_file=None, delay_saves=
   get_ctrl().reinit(settings, change_notifier=notify_on_change)
   return app
 
+def advertise_service(port):
+  """ Advertise the AmpliPi api via zerconf, can be verified with 'avahi-browse -ar' """
+  hostname = f'{gethostname()}.local'
+  url = f'http://{hostname}'
+  if port != 80:
+    url += f':{port}'
+  info = ServiceInfo(
+    "_http._tcp.local.",
+    "amplipi-api._http._tcp.local.", # this is named AmpliPi-api to distinguish from the common Spotify/Airport name of AmpliPi
+    addresses=[inet_aton("127.0.0.1")],
```

Comment:
I'm seeing the "address" field advertised as 127.0.0.1 even when on another computer. I assume this should be an appropriate IP for each interface? E.g. I'd expect to see 127.0.0.1 on `lo` but not on `eth0`, nor on another computer at all.

Reply:
Hmm, I'll check into this. I wonder what is advertised if that field is not populated.